tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -1094,6 +1098,70 @@ const reducer = createReducer(
       singleSelectionHeaders: newOrder,
     };
   }),
+  on(actions.dataTableColumnEdited, (state, {fobState, newHeaders}) => {
+    const enabledNewHeaders: ColumnHeader[] = [];
+    const disabledNewHeaders: ColumnHeader[] = [];
+
+    newHeaders.forEach((header) => {
```

Comment:
A little comment explaining that all enabled headers appear above all disabled headers would be helpful.

Reply:
Added.